.github/workflows/release.yaml

```
@@ -24,7 +24,7 @@ env:
   PULP_URL: "https://download.konghq.com/kuma-binaries-release/"
   DOCKER_IMAGES: "kumactl,kuma-cp,kuma-dp,kuma-init,kuma-cni"
   RELEASE: ${{ inputs.release || '0.0.0' }}
-  CHECK: ${{ inputs.check === "true" }}
+  CHECK: ${{ inputs.check || "true" }}
```

```suggestion
  CHECK: ${{ inputs.check == "true" }}
```

sorry, I mixed JS in. Shouldn't be `|| true` either way I don't think

---

This still doesn't work: https://github.com/lahabana/sandbox/actions/runs/4666440773

---

that's double vs single quotes